### PR TITLE
Create google_recaptcha_api.js

### PR DESCRIPTION
**Pull request by mrahla**

### CODE REVIEW

**Comment by Dominator008** [comment]
Thanks, please sign the CLA and format the file according to Google's JavaScript Style Guide (https://google.github.io/styleguide/javascriptguide.xml)

File: contrib/externs/google_recaptcha_api.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+
```
Comment (Dominator008) [inline contrib/externs/google_recaptcha_api.js]: Remove extra line.

File: contrib/externs/google_recaptcha_api.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+ * 
+ * @fileoverview Externs for the Google Recaptcha v2 Public API (api.js).
+ * @see https://developers.google.com/recaptcha/docs/display
+ * @externs
+ */
+
+/**
+ * @constructor
+ */
+
```
Comment (Dominator008) [inline contrib/externs/google_recaptcha_api.js]: Remove extra line.

File: contrib/externs/google_recaptcha_api.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+function grecaptcha() { }
```
Comment (Dominator008) [inline contrib/externs/google_recaptcha_api.js]: function grecaptcha() {}

File: contrib/externs/google_recaptcha_api.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+
+/**
+ * @param {(string|Element)} container
+ * @param {Array.<string>=} parameters
+ * @return {number}
+ */
+grecaptcha.prototype.render = function (container, parameters) { };
```
Comment (Dominator008) [inline contrib/externs/google_recaptcha_api.js]: grecaptcha.prototype.render = function(container, parameters) {};

File: contrib/externs/google_recaptcha_api.js [hunk 5]
```diff
@@ -0,0 +1,37 @@
+
+/**
+ * Copyright 2016 ZapGroup LTD
```
Comment (MatrixFrog) [inline contrib/externs/google_recaptcha_api.js]: I think we need the copyright line to say "The Closure Compiler Authors"

Comment (Dominator008) [inline contrib/externs/google_recaptcha_api.js]: We do have things like "YouTube LLC" :)
https://github.com/google/closure-compiler/blob/master/contrib/externs/youtubeplayer.js

But yeah, "The Closure Compiler Authors" might be better.

**Comment by dimvar** [comment]
Closing due to inactivity. Please comment if you would like to finish this, we can reopen.

**Pull request closed without merging**